atm.js

```diff
@@ -1,3 +1,115 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+
+var checkDeposit = $("#checking .deposit");
+var checkWithdraw = $("#checking .withdraw");
+var savingDeposit = $("#savings .deposit");
+var savingWithdraw = $("#savings .withdraw");
+var inputChecking = $("#checking .input");
+var inputSaving = $("#savings .input");
+var storedChecking = $("#checking .balance");
+var storedSaving = $("#savings .balance");
+
+checkDeposit.on("click", function(evt){
+ evt.preventDefault();
+ var amount = $("#checking .input").val();
+ $("#checking .balance").html("$" + amount);
+})
```

Comment:
i think you're really close, really just have to add the amount to what already exists in the balance.